src/components/common/Extensions/MainNavigation/MainNavigation.tsx

```diff
@@ -1,42 +1,103 @@
 import React, { FC } from 'react';
 
+import { useIntl } from 'react-intl';
```

Comment:
Move the import package to the top (imports packages and other packages like hooks, types,components etc. should be separate by an empty line).

Reply:
Fixed